src/create-tree.ts

```diff
@@ -71,7 +95,7 @@ export async function createTree(
         result = await value({ exists: false });
       }
 
-      if (result === null || typeof result === "undefined") continue;
+      if (result === null || typeof result === "undefined" || typeof result === "symbol") continue;
       tree.push(
         await valueToTreeObject(octokit, ownerOrFork, repo, path, result)
```

Comment:
TS says:
```
Argument of type 'string | File | Symbol' is not assignable to parameter of type 'string | File'.
Type 'Symbol' is not assignable to type 'string | File'
```

BUT, the condition above prevents this code path if result is of type symbol or am I missing something here?

Reply:
> the condition above prevents this code path if result is of type symbol or am I missing something here

if that happenes and TS is not smart enough to get it, I'd suggest to add `// @ts-expect-error - explanation here` to make TS calm down about it